src/tokenized-line.coffee

```diff
@@ -59,7 +74,7 @@ class TokenizedLine
     screenColumn = 0
     currentBufferColumn = 0
     for token in @tokens
-      break if currentBufferColumn > bufferColumn
+      break if currentBufferColumn + token.bufferDelta > bufferColumn
```

Comment:
Ideally, we should have a failing test for each change we make to the code: deleting the `+ token.bufferDelta`, however, has no impact on the test suite.

Could you explain why you changed this, please? Also, could you add a test for it?